package.json

```diff
@@ -12,12 +16,18 @@
     "autoprefixer": "10.4.14",
     "eslint": "8.36.0",
     "eslint-config-next": "13.2.4",
+    "eslint-plugin-jest-dom": "^5.1.0",
+    "eslint-plugin-testing-library": "^6.0.1",
     "fluent-ffmpeg": "2.1.2",
+    "jest": "^29.7.0",
+    "jest-dom": "^4.0.0",
```

Comment:
Apparently I shouldn't use this one, but the testing-library version instead. let me try that